python/numpy/Exercise-HPLC.ipynb

```diff
@@ -138,14 +140,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Plot intensities\n",
-    "import matplotlib.colors as colors\n",
-    "\n",
+    "# Plot the intensities\n",
     "fig = plt.figure()\n",
-    "plt.imshow(intensities, norm=colors.LogNorm(), aspect=\"auto\")\n",
-    "\n",
-    "# Note: with latest version of matplotlib:\n",
-    "# plt.imshow(intensities, norm=\"log\", aspect=\"auto\")"
+    "plt.imshow(intensities, norm=\"log\", aspect=\"auto\")"
```

Comment:
I think this does not work with the environment available in jupyter-slurm, did you tested there?